src/pages/FavouriteSchoolsPage.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import Layout from '../components/Layout';
+import Container from '../components/Container';
+import Breadcrumbs from '../components/Breadcrumbs';
+import PageTitle from '../components/PageTitle';
+import { getFavSchoolsFromStorage } from '../utils/localStorageFavSchools';
+import FavouriteSchool from '../components/sections/FavouriteSchools/FavouriteSchool';
+import NoFavSchoolsInfo from '../components/sections/FavouriteSchools/NoFavSchoolsInfo';
+
+const FavouriteSchoolsPage: React.FC<RouteComponentProps> = () => {
+  const [updateKey, setKey] = useState<number>(0);
+
+  const favouriteSchoolsIDs: string[] = getFavSchoolsFromStorage();
```

Comment:
I noticed that by using `updateKey` you force component to rerender and that's how you keep `favouriteSchoolsIDs` up-to-date. 
I would suggest to rewrite a bit `useFavouritesSchools` hook so it can keep ids of favourite schools somehow reactive. Then you would be able to use this hook to pass to each element something like `removeSchoolFromFavourites(schoolID)` via props.

Reply:
I changed the behaviour of this hook